serialize.js

```diff
@@ -104,3 +111,14 @@ function properties(elem) {
 
     return props.length ? stringify(props) : ""
 }
+
+function escapeText(str) {
```

Comment:
Well, prior to this patch it was possible to do something like

```JavaScript
var div = document.createElement('div');
div.textContent = '<p>text</p>'; // should treat as text, not html
div.setAttribute('class', '"')

div.toString() // wrongly outputs '<div class="""><p>text</p></div>
```

Or do you mean something else was handling this previously?